app/components/application_record_card/view.rb

```diff
@@ -67,6 +66,14 @@ def trn
       tag.p("TRN: #{record.trn}", class: "govuk-caption-m govuk-!-font-size-16 application-record-card__trn govuk-!-margin-bottom-0 govuk-!-margin-top-0")
     end
 
+    def show_provider
```

Comment:
Do we have a consensus on whether this type of check should be a policy thing? Just wondering why we've moved the `hide_progress_tag` into the `TraineePolicy` but not this one.